hs_core/urls.py

```diff
@@ -12,15 +13,32 @@
     url(r'^groups/$', views.users_api.CreateOrListGroups.as_view()),
     url(r'^groups/(?P<pk>[A-z0-9]+)/$', views.users_api.ListGroupMembers.as_view()),
     url(r'^groups/(?P<g>[A-z0-9]+)/owner/(?P<u>[A-z0-9]+)/$', views.users_api.SetOrDeleteGroupOwner.as_view()),
-    url(r'^resources/$', views.users_api.GetResourceList.as_view()),
+    #url(r'^resources/$', views.users_api.GetResourceList.as_view()),
+
+    # new user authenticate REST API (API for testing) based on django rest framework
+    url(r'^authenticate/$', views.user_rest_api.authenticate_user, name='authenticate'),
```

Comment:
Need to document this end point in the (spec)[https://github.com/hydroshare/hydroshare/blob/develop/docs/DesignDocs/API/index.md].